src/test/v2.0/advanced/fdc3.raiseIntent-NoAppsFound.ts

```diff
@@ -0,0 +1,101 @@
+import { ResolveError } from "fdc3_2_0";
+import { assert, expect } from "chai";
+import { DesktopAgent } from "fdc3_2_0/dist/api/DesktopAgent";
+import { APIDocumentation2_0 } from "../apiDocuments-2.0";
+import { RaiseIntentControl2_0 } from "./intent-support-2.0";
+import constants from "../../../constants";
+import { sleep, wrapPromise } from "../../../utils";
+
+declare let fdc3: DesktopAgent;
+const control = new RaiseIntentControl2_0();
+const raiseIntentDocs = "\r\nDocumentation: " + APIDocumentation2_0.raiseIntent + "\r\nCause";
+
+/**
+ * Details on the mock apps used in these tests can be found in /mock/README.md
+ */
+export default () =>
+  describe("fdc3.raiseIntent (NoAppsFound)", () => {
+    const RaiseIntentFailedResolve = "(RaiseIntentFailedResolve) Should fail to raise intent when targeted app intent-a, context 'testContextY' and intent 'aTestingIntent' do not correlate";
+    it(RaiseIntentFailedResolve, async () => {
+      try {
+        await control.raiseIntent("aTestingIntent", "testContextY");
+        assert.fail("No error was thrown");
+      } catch (ex) {
+        expect(ex, raiseIntentDocs).to.have.property("message", ResolveError.NoAppsFound);
+      }
+    });
+
+    const RaiseIntentFailTargetedAppInstanceResolve1 =
+      "(RaiseIntentFailTargetedAppInstanceResolve1) Should fail to raise intent when targeted app intent-a instance, context 'testContextY', intent 'aTestingIntent' and AppIdentifier IntentAppAId do not correlate";
+    it(RaiseIntentFailTargetedAppInstanceResolve1, async () => {
+      try {
+        const appIdentifier = await control.openIntentApp("IntentAppAId");
+        await control.raiseIntent("aTestingIntent", "testContextY", appIdentifier);
+        assert.fail("No error was thrown");
+      } catch (ex) {
+        expect(ex, raiseIntentDocs).to.have.property("message", ResolveError.NoAppsFound);
```

Comment:
I got this wrong in the test cases, it should be:
```suggestion
        expect(ex, raiseIntentDocs).to.have.property("message", ResolveError.ResolveError.IntentDeliveryFailed);
```

Suggested correction of the test case: https://github.com/finos/FDC3/pull/857/files#r1049935861